test/src/test_derive.rs

```diff
@@ -178,6 +192,236 @@ fn test_derive_owned_to_variant() -> bool {
     ok
 }
 
+#[derive(NativeClass)]
+#[inherit(Reference)]
+struct MinimalDerive(i64);
+
+#[methods]
+impl MinimalDerive {
+    fn new(_owner: &Reference) -> Self {
+        Self(54)
+    }
+
+    #[export]
+    fn answer(&self, _owner: &Reference) -> i64 {
+        self.0
+    }
+}
+
+fn test_derive_nativeclass() -> bool {
+    println!(" -- test_derive_nativeclass");
+
+    let ok = std::panic::catch_unwind(|| {
+        let thing = Instance::<MinimalDerive, _>::new();
+        let base = thing.into_base();
+        assert_eq!(unsafe { base.call("answer", &[]).to::<i64>() }, Some(54),);
```

Comment:
Trailing comma after `Some(54)`, I guess rustfmt doesn't check the macro invocations 🤔